netshare/generate/model_generate.py

```diff
@@ -20,42 +20,56 @@
 def model_generate() -> None:
     configs, config_group_list = create_chunks_configurations(generation_flag=True)
 
-    logger.info("Start generating attributes")
-    ray.get(
-        [
-            generate_data.remote(
-                config=config,
-                given_data_attribute_flag=False,
-            )
-            for config in configs
-        ]
-    )
+    # single chunk case
+    if len(config_group_list[0]["config_ids"]) == 1:
```

Comment:
```suggestion
    if get_config("global_config.n_chunks", default_value=1) == 1:
```
Although they both similar, we're making the reader understand where this is come from